app/src/sharedTest/java/org/oppia/android/app/story/StoryFragmentTest.kt

```diff
@@ -489,6 +536,59 @@ class StoryFragmentTest {
     }
   }
 
+  /**
+   * Reference:
```

Comment:
check `SegmentedCircularProgressView` for the doc format

Reply:
I previously used this format `Reference:    //  https://stackoverflow.com/questions/38314077/how-to-click-a-clickablespan-using-espresso` but it was exceeding the max character length. 

We can use the shortened URL of the question as `Reference:    //  https://stackoverflow.com/q/38314077` 

@anandwana001 please let me know your thoughts on this.

Thanks.

Reply:
I think the max length not work on comments. Does it?

Reply:
I am actually not sure about this.

Reply:
you can try it.
1. Keep it in one line
2. run the script - `bash scripts/ktlint_lint_check.sh` from you Android Studio terminal. If using windows, try to run it from the git bash command line. 
3. If you don't want to run the script, simple keep change to one line, commit it and try to push, the error will shown in command line if it occurs.

Reply:
okay.